validator/docs/schemas.md

```diff
@@ -0,0 +1,163 @@
+# Configuration Validator
+
+This module contains the logic needed to validate deployment files with a defined schema by Nestor.
+
+The deployment files define applications or projects.
+
+## Application Configuration
+
+In each file you will define the application that you want to deploy, there are few fields which are required to setup a new application.
+
+This is a basic configuration application with the minimum required fields.
+
+```yaml
+app: my-application   # Application's name
+is_enabled: false     # Is application deployment enabled
+git: git@microservice_repo # Git repository URL
+variables:            # Environment variables
+  app:
+    APP_VAR1: application_variable_1
+  ope:
+    OPE_VAR1: ope_variable_1
+```
+
+To customize your application deployment, there are other sections supported. Check the following sections.
+
+### Processes
+
+Define processes of a project. A project can contain multiple processes such as workers, web processes and more.
+The structure to define a process is the following:
+
+```yaml
+processes:
+  - name: process-name  # Process name, STRING - Required
```

Comment:
I believe that we have a limit on the process name length, it could be useful to document it here